.github/workflows/tests.yml

```diff
@@ -31,6 +31,7 @@ jobs:
 
   integration_tests:
     runs-on: ubuntu-latest
+    needs: [linter]
```

Comment:
why needs linter?
We would like to see if tests are failing or not even if there is any issues in the linter test

Reply:
Tests started when the linter passed, our requirement linter passed, and the user fixed the linter issue first.